docs/content/en/docs/reference/packagespec/metallb/v0.12.1.md

```diff
@@ -0,0 +1,43 @@
+| **peer** |||
+| peer-address | The target IP address for the BGP session ||
+| peer-asn | The BGP AS number that MetalLB expects to see advertised by the router ||
+| my-asn | The BGP AS number that MetalLB should speak as ||
+| peer-port | (optional) the TCP port to talk to | 179 |
+| source-address | (optional) The source IP address to use when establishing the BGP session. The address must be configured on a local network interface ||
+| hold-time | (optional) The proposed value of the BGP Hold Time timer. Refer to BGP reference material to understand what setting this implies ||
+| keepalive-time |  (optional) The keepalive interval to be used in the BGP session |hold-time / 3|
+| router-id | (optional) The router ID to use when connecting to this peer. | Node IP |
+| password | (optional) Password for TCPMD5 authenticated BGP sessions offered by some peers ||
+| ebgp-multihop | (optional) Whether eBGP multihop is permitted. Note that it is always on in the native BGP mode ||
+| node-selectors | (optional) The nodes that should connect to this peer. A node matches if at least one of the node selectors matches. Within one selector, a node matches if all the matchers are satisfied. The semantics of each selector are the same as the label- and set-based selectors in Kubernetes, documented at https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/. By default, all nodes are selected.<br/><pre>node-selectors:<br/># Match by label=value<br/>  - match-labels:<br/>    kubernetes.io/hostname: prod-01<br/># Match by 'key OP values' expressions<br/>  - match-expressions:<br/>    key: beta.kubernetes.io/arch<br/>    operator: In<br/>    values: [amd64, arm]</pre>||
+| **bgp-advertisement** |||
+|aggregation-length| (optional) How much you want to aggregate up the IP address before advertising. For example, advertising 1.2.3.4 with aggregation-length=24 would end up advertising 1.2.3.0/24. For the majority of setups, you'll want to keep this at the default of 32, which advertises the entire IP address unmodified | 32 |
+|aggregation-length-v6| (optional) How much you want to aggregate up the IPv6 address before advertising. For example, advertising 2001:0db8:85a3:0000:0000:8a2e:0370:7334 with aggregation-length-v6=64 would end up advertising 2001:0db8:85a3:0000:0000:0000:0000:0000/64. For the majority of setups, you'll want to keep this at the default of 128, which advertises the entire IP address unmodified|128|
+|localpref| (optional) The value of the BGP "local preference" attribute for this advertisement. Only used with IBGP peers, i.e. peers where peer-asn is the same as my-asn. ||
+|communities[]| (optional) BGP communities to attach to this advertisement. Communities are given in the standard two-part form <asn>:<community number>. You can also use alias names ||
```

Comment:
```suggestion
|communities[]| (optional) BGP communities to attach to this advertisement. Communities are given in the standard two-part form \<asn\>:\<community number\>. You can also use alias names ||
```
It sees the greater-than and less-than signs as special characters. Check if just adding backslashes before each of those characters works.